.github/workflows/full_cd.yaml

```diff
@@ -0,0 +1,73 @@
+name: Deploy Query Connector to demo site
+
+on:
+  merge_group:
+    types:
+      - checks_requested
+  push:
+    branches:
+      - main
+      - shanice/automated-workflow-deployment
```

Comment:
Is your personal branch intended to be here?

Reply:
Used for testing purposes, removed.